pkg/kubeapiserver/default_storage_factory_builder.go

```diff
@@ -89,6 +90,7 @@ func NewStorageFactoryConfigEffectiveVersion(effectiveVersion basecompatibility.
 		certificates.Resource("clustertrustbundles").WithVersion("v1beta1"),
 		storage.Resource("volumeattributesclasses").WithVersion("v1beta1"),
 		storagemigration.Resource("storagemigrations").WithVersion("v1alpha1"),
+		resource.Resource("devicetaintrules").WithVersion("v1alpha3"),
```

Comment:
Why `v1alpha3` and not `v1alpha1`?

Reply:
Because that is the current alpha for the resource.k8s.io API group.

Bringing back a v1alpha1 and then adding the type there in addition to v1alpha3 and v1beta1 doesn't make sense.